app/controllers/admin/routes.py

```diff
@@ -430,12 +430,17 @@ def updatecohort(year, method, username):
     if method == "add":
         try:
             BonnerCohort.create(year=year, user=user)
+            flash(f"Successfully added {user.fullName} to {year} Bonner Cohort", "success")
         except IntegrityError as e:
             # if they already exist, ignore the error
+            flash(f'Error: {user.fullName} already added.', "danger")
```

Comment:
It is personal preference but I would prefer to take "Error:" out of the error flash when the person is already in a cohort so it would be "User already in cohort.".

I am also not a big fain of "Error:" in the second error flash but it does fit a bit better I think. I think I would prefer "Failed to add User." but again, it is personal preference.

Reply:
Corrected punctuation and made requested change.